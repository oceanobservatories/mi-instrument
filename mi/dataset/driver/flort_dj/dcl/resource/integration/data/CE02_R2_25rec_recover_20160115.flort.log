2016/01/15 00:00:13.530 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2016/01/15 23:47:31.213 01/15/16	23:43:54	700	201	695	133	460	65	551
2016/01/15 23:47:32.343 01/15/16	23:43:55	700	198	695	133	460	65	551
2016/01/15 23:47:33.472 01/15/16	23:43:57	700	200	695	129	460	67	551
2016/01/15 23:47:34.602 01/15/16	23:43:58	700	233	695	129	460	66	551
2016/01/15 23:47:35.732 01/15/16	23:43:59	700	209	695	131	460	67	551
2016/01/15 23:47:36.861 01/15/16	23:44:00	700	263	695	128	460	66	550
2016/01/15 23:47:37.990 01/15/16	23:44:01	700	201	695	125	460	68	551
2016/01/15 23:47:39.119 01/15/16	23:44:02	700	191	695	128	460	64	550
2016/01/15 23:47:40.248 01/15/16	23:44:03	700	232	695	128	460	67	550
2016/01/15 23:47:41.378 01/15/16	23:44:04	700	213	695	131	460	67	550
2016/01/15 23:47:42.507 01/15/16	23:44:06	700	224	695	128	460	69	550
2016/01/15 23:47:43.636 01/15/16	23:44:07	700	211	695	131	460	68	550
2016/01/15 23:47:44.766 01/15/16	23:44:08	700	196	695	132	460	74	550
2016/01/15 23:47:45.894 01/15/16	23:44:09	700	228	695	127	460	67	550
2016/01/15 23:47:47.024 01/15/16	23:44:10	700	225	695	131	460	67	550
2016/01/15 23:47:48.154 01/15/16	23:44:11	700	209	695	136	460	66	550
2016/01/15 23:47:49.283 01/15/16	23:44:12	700	208	695	132	460	66	550
2016/01/15 23:47:50.413 01/15/16	23:44:13	700	206	695	128	460	66	550
2016/01/15 23:47:51.542 01/15/16	23:44:15	700	204	695	130	460	64	550
2016/01/15 23:47:52.680 01/15/16	23:44:16	700	210	695	130	460	66	550
2016/01/15 23:47:53.801 01/15/16	23:44:17	700	188	695	129	460	65	550
2016/01/15 23:47:54.931 01/15/16	23:44:18	700	234	695	129	460	66	550
2016/01/15 23:47:56.060 01/15/16	23:44:19	700	210	695	131	460	65	550
2016/01/15 23:47:57.190 01/15/16	23:44:20	700	213	695	141	460	67	550
2016/01/15 23:47:58.319 01/15/16	23:44:21	700	262	695	150	460	66	550
2016/01/15 23:48:00.796 [flort:DLOGP2]:Instrument Stopped [Power Off]
