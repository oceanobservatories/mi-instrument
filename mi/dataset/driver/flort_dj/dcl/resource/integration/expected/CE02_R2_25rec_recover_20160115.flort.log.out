
 subsite  | node  | sensor       | bin        | method    | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-----------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890451.2130002975 |          2 |         3661890234 | 3661890451.2130002975 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               551 |             201 |              65 |            133 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890452.3429999352 |          2 |         3661890235 | 3661890452.3429999352 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               551 |             198 |              65 |            133 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890453.4720001221 |          2 |         3661890237 | 3661890453.4720001221 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               551 |             200 |              67 |            129 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890454.6020002365 |          2 |         3661890238 | 3661890454.6020002365 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               551 |             233 |              66 |            129 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered |  3661890455.732000351 |          2 |         3661890239 |  3661890455.732000351 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               551 |             209 |              67 |            131 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered |  3661890456.861000061 |          2 |         3661890240 |  3661890456.861000061 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             263 |              66 |            128 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890457.9899997711 |          2 |         3661890241 | 3661890457.9899997711 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               551 |             201 |              68 |            125 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered |  3661890459.118999958 |          2 |         3661890242 |  3661890459.118999958 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             191 |              64 |            128 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered |  3661890460.248000145 |          2 |         3661890243 |  3661890460.248000145 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             232 |              67 |            128 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890461.3780002594 |          2 |         3661890244 | 3661890461.3780002594 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             213 |              67 |            131 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890462.5069999695 |          2 |         3661890246 | 3661890462.5069999695 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             224 |              69 |            128 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890463.6359996796 |          2 |         3661890247 | 3661890463.6359996796 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             211 |              68 |            131 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered |  3661890464.765999794 |          2 |         3661890248 |  3661890464.765999794 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             196 |              74 |            132 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890465.8940000534 |          2 |         3661890249 | 3661890465.8940000534 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             228 |              67 |            127 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890467.0240001678 |          2 |         3661890250 | 3661890467.0240001678 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             225 |              67 |            131 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890468.1540002823 |          2 |         3661890251 | 3661890468.1540002823 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             209 |              66 |            136 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890469.2829999924 |          2 |         3661890252 | 3661890469.2829999924 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             208 |              66 |            132 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890470.4130001068 |          2 |         3661890253 | 3661890470.4130001068 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             206 |              66 |            128 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890471.5419998169 |          2 |         3661890255 | 3661890471.5419998169 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             204 |              64 |            130 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890472.6800003052 |          2 |         3661890256 | 3661890472.6800003052 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             210 |              66 |            130 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890473.8010001183 |          2 |         3661890257 | 3661890473.8010001183 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             188 |              65 |            129 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890474.9309997559 |          2 |         3661890258 | 3661890474.9309997559 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             234 |              66 |            129 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890476.0599999428 |          2 |         3661890259 | 3661890476.0599999428 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             210 |              65 |            131 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890477.1900000572 |          2 |         3661890260 | 3661890477.1900000572 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             213 |              67 |            141 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3661804800 | recovered | 3661890478.3190002441 |          2 |         3661890261 | 3661890478.3190002441 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               550 |             262 |              66 |            150 |        null

(25 rows)
